code/modules/power/supermatter/supermatter.dm

```diff
@@ -1049,7 +1094,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 			investigate_log("has consumed [consumed_object] - [suspicion].", INVESTIGATE_SUPERMATTER)
 		qdel(consumed_object)
 	if(!iseffect(consumed_object) && power_changes)
-		matter_power += 200
+		matter_power += 200 * antinoblium_multiplier
```

Comment:
You should make an add_matter_power proc so you don't need to remember to copypasta this onto everything that effects it

Reply:
Should I make the proc itself get multiplied by that, or should it be a parameter?

Reply:
Alright, does my add_matter_power_proc commit do?

Reply:
Mostly. if you want to be really clean about it make the proc return the amount added, and use that number to deal with miasma